### PR TITLE
add helm install in readme

### PR DESCRIPTION
**Pull request by lowang-bh**
1. Install official release for end user
2. Install from source code for developer

### CODE REVIEW

**Comment by wangyang0616** [comment]
I think two things should be described here:

1. Helm deploys the official release version of Volcano
    The manual has been submitted in the helm-charts warehouse [Add usage of Volcano helm repo to Readme #1](https://github.com/volcano-sh/helm-charts/pull/1), here you can make a link to the Readme of the helm-charts warehouse
2. Developers use helm to install Volcano through source code

**Comment by lowang-bh** [comment]
> I think two things should be described here:
> 
> 1. Helm deploys the official release version of Volcano
>    The manual has been submitted in the helm-charts warehouse [Add usage of Volcano helm repo to Readme #1](https://github.com/volcano-sh/helm-charts/pull/1), here you can make a link to the Readme of the helm-charts warehouse
> 2. Developers use helm to install Volcano through source code

Thanks for you suggestion. How about the latest one?

File: README.md [hunk 1]
```diff
@@ -107,6 +107,24 @@ job.batch/volcano-admission-init   1/1           48s        96s
 
 ```
 
+### Install via helm
+
+To install official release, please visit to https://github.com/volcano-sh/helm-charts for details.
```
Comment (wangyang0616) [inline README.md:112]: Is it better to update `https://github.com/volcano-sh/helm-charts` to [helm-charts](`https://github.com/volcano-sh/helm-charts`)

**Comment by wangyang0616** [comment]
> Thanks for you suggestion. How about the latest one?
Thank you very much for your update, there is no problem with the content, and there is a small problem with the hyperlink format, which has been commented

File: README.md (hunk 1 above)
Comment (lowang-bh) [reply README.md]: Done.

**Comment by wangyang0616** [comment]
It's ok to me. Please help review. Thanks! @william-wang

**Review by hwdef — COMMENTED** [review]
/lgtm

**Review by william-wang — APPROVED** [review]
/lgtm

**Pull request merged**